### ISSUE TITLE
English abstract does not follow correct hyphenation rules

### ISSUE THREAD

**Issue by gszasz** [sars1492/diplomovka#14]
The Slovak language has different hyphenation rules, so they cannot be applied on text written in English. 

## Suggested Fix

The babel package is capable to temporarily switch to different language when treating paragraph actually written in different language. This feature is needs to be implemented into the thesis template to to follow correct hyphenation rules for English abstract.

```latex
\usepackage[english,slovak]{babel} % the last language is the default one
...
\begin{otherlanguage}{english}  % switching temporarily to English
Abstract
...
\end{otherlanguage}
```

**Comment by gszasz** [comment]
The current version of English abstract doesn't need any hyphenation and thus this bug won't be fixed in the milestone 0.1.

**Comment by gszasz** [comment]
The bug was fixed by pull request #20.
